### DISCUSSION TITLE
[Bug]: Themepatcher doesn't work

### DISCUSSION THREAD

**Discussion by drnikos** [prasanthrangan/hyprdots#1239] [General]
[hyde] Whenever I try to patch an extra theme, even during installation, I get an error that kvantum.theme doesnt exist. I didn't have this issue in previous weeks.
![240403_15h13m16s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/153459342/d2f65bad-857c-48db-856f-f9ef9a122c7f)
Any ideas?

**Comment by kRHYME7** [comment]
The Theme Repository should adapt to the new theme structure. A repo that is compatible of the Hyde branch for now is this repo https://github.com/kRHYME7/Hyprdots-gallery. 

I'll move this to discussion as you are actually using the dev branch that pushes changes frequently. 

And this matter is relevant to be discussed at our Discord server https://discord.gg/FjkBSNGm
